packages/react/src/Button/ButtonBase.module.css

```diff
@@ -430,8 +430,6 @@
     &:disabled,
     &[aria-disabled='true']:not([data-loading='true']) {
       color: var(--button-invisible-fgColor-disabled);
```

Comment:
Comparison of rest/disabled colors for reference:

<img width="253" alt="screenshot of enabled/disabled invisible iconbuttons" src="https://github.com/user-attachments/assets/0cb8413d-70a9-49e1-bc76-5818ee38004e" />